core-api/src/main/java/com/optimizely/ab/Optimizely.java

```diff
@@ -454,6 +456,19 @@ public Boolean getFeatureVariableBoolean(@Nonnull String featureKey,
         );
     }
 
+    /**
+     * Get {@link OptimizelyConfig} containing experiments and features map
+     *
+     * @return {@link OptimizelyConfig}
+     */
+    public OptimizelyConfig getOptimizelyConfig() {        
```

Comment:
I see this new API is located in the middle of getVariable section. Can we add at the end of other APIs as a separate section?